packages/shared/src/validators/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./managed-by";
```

Comment:
It's a little annoying that we have to update this index file every time we add a new validator.